app/frontend/src/features/profile/view/leaveReference/formSteps/submit/ReferenceOverview.tsx

```diff
@@ -46,19 +46,13 @@ export default function ReferenceOverview({
         {PRIVATE_REFERENCE}
       </Typography>
       <ul>
-        {referenceData.wasAppropriate === "true" ? (
-          <li>
-            <TextBody className={classes.text}>
-              {COUCHER_WAS_APPROPRIATE}
-            </TextBody>
-          </li>
-        ) : (
-          <li>
-            <TextBody className={classes.text}>
-              {COUCHER_WAS_NOT_APPROPRIATE}
-            </TextBody>
-          </li>
-        )}
+        <li>
+          <TextBody className={classes.text}>
+            {referenceData.wasAppropriate === "true"
```

Comment:
```suggestion
            {referenceData.wasAppropriate
```
Right? There should be a lint warning here saying this will never be true because of type mismatch